src/review/dto/create-review.dto.ts

```diff
@@ -18,7 +19,7 @@ export class CreateReviewDto {
   @ApiProperty({ description: '음식 id' })
   foodId: string;
 
-  @IsString({ each: true })
-  @ApiProperty({ description: '음식 태그 id' })
-  tagIds: string[];
+  @IsEnum(TASTE_TAG, {each: true})
+  @ApiProperty({ description: "음식 태그 ex) '얼얼한', '칼칼한', '매콤달콤한', '알싸한', '얼큰한', '개운한'", })
+  tagIds: TASTE_TAG[];
```

Comment:
더이상 id를 받는 게 아니라 enum string을 받으니까 그냥 아래처럼 하는게 좋지 않을까요?
```suggestion
  @IsEnum(TASTE_TAG, {each: true})
  @ApiProperty({ description: "음식 태그 ex) '얼얼한', '칼칼한', '매콤달콤한', '알싸한', '얼큰한', '개운한'", })
  tags: TASTE_TAG[];
```